src/config.ts

```diff
@@ -325,7 +325,7 @@ export const BONUS_API_CLIENT = BonusAPIClient(
   httpApiFetch
 );
 // the date until a user can request a bonus
-export const BONUS_REQUEST_LIMIT_DATE = UTCISODateFromString.decode(
+export const BONUS_REQUEST_LIMIT_DATE = DateFromString.decode(
```

Comment:
Why `DateFromString` and not a timestamp? The timestamp must be included inside another ENV variable es. `BONUS_REQUEST_LIMIT_TIMESTAMP`

Reply:
We don't have a ready-to-use timestamp decoder. `DateFromString` actually handles a broader set of formats as it's just _any strings that can be interpreted as date_: https://github.com/pagopa/io-ts-commons/blob/edb31883fa5e7c9bb21f0722f000321b6554e37c/src/dates.ts#L10.

Actually, the set of possibles formats is inherited by the js `Date` constructor: https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Global_Objects/Date.

Now I realise we might need more test on that:
```js
[
  1577836800000,
  "1577836800000", 
  "2020",
  2020,
  "2018-07-10T13:53:03.987Z"
].forEach(t => console.log(new Date(t)))
/*
Wed Jan 01 2020 01:00:00 GMT+0100 (Central European Standard Time)
Invalid Date
Wed Jan 01 2020 01:00:00 GMT+0100 (Central European Standard Time)
Thu Jan 01 1970 01:00:02 GMT+0100 (Central European Standard Time)
Tue Jul 10 2018 15:53:03 GMT+0200 (Central European Summer Time)
*/
```

Reply:
DateFromString seems cannot work with a timestamp. Because require a string decode before passing the parameter to the `Date` constructor ([ref](https://github.com/pagopa/io-ts-commons/blob/edb31883fa5e7c9bb21f0722f000321b6554e37c/src/dates.ts#L16))

Reply:
We could create a new decoder
```typescript
export const DateFromTimestamp = new t.Type<Date, number>(
  "DateFromString",
  isDate,
  (v, c) =>
    isDate(v)
      ? t.success(v)
      : t.number.validate(v, c).chain(s => {
          const d = new Date(s);
          return isNaN(d.getTime()) ? t.failure(s, c) : t.success(d);
        }),
  a => a.valueOf()
);
```
and using it like 
```typescript
NumberFromString.decode(a)
  .chain(DateFromTimestamp.decode)
  .map(a => console.log(a));
```